docs/src/components/landing/DemoVideo.js

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+
+const Video = styled('video')(({ theme }) => ({
+  borderStyle: 'solid',
+  borderColor:
+    theme.palette.mode === 'dark' ? theme.palette.primaryDark[900] : theme.palette.grey[100],
+  borderRadius: '10px 10px 0 0 ',
+  overflow: 'hidden',
+  objectFit: 'cover',
+  objectPosition: 'top',
+  outline: `1px solid ${
+    theme.palette.mode === 'dark' ? theme.palette.primaryDark[400] : theme.palette.grey[300]
+  }`,
```

Comment:
Note that Safari doesn't support `outline` border-radius. It looks like this:

<img width="1194" alt="Screenshot 2022-08-25 at 12 49 12" src="https://user-images.githubusercontent.com/3165635/186645551-9af0607d-40aa-4526-b7cc-98440f21d357.png">